bin/dfx-canister-url

```diff
@@ -97,13 +99,15 @@ done
 
 # Some canisters have custom URLs, such as identity.ic0.app.
 if [[ "${DFX_URL_TYPE:-}" == "static" ]]; then
-  url="$(jq -r '.canisters[env.DFX_CANISTER].url[env.DFX_NETWORK] // ""' dfx.json)"
+  key="${DFX_RAW:+raw_url}"
+  key="${key:-url}"
```

Comment:
To me these 2 lines are (if I even understood them correctly) a very unreadable way of writing
```
if [ "$DFX_RAW" ]; then
  key="raw_url"
else
  key="url"
fi
```
Even something like this (assuming that it works) would be a lot more readable to me:
```
[ "$DFX_RAW" ] && key="raw_url" || key="url"
```